config/IpCfg.hpp

```diff
@@ -19,10 +19,9 @@ enum IpCfg {
     SOCKET_IP_SEND_FLAGS = 0,              // send, sendto FLAGS argument
     SOCKET_IP_RECV_FLAGS = 0,              // recv FLAGS argument
     SOCKET_MAX_ITERATIONS = 0xFFFF,        // Maximum send/recv attempts before an error is returned
-    SOCKET_RETRY_INTERVAL_MS = 1000,       // Interval between connection retries before main recv thread starts
     SOCKET_MAX_HOSTNAME_SIZE = 256         // Maximum stored hostname
 };
-
+const Fw::Time SOCKET_RETRY_INTERVAL = Fw::Time(1, 0);
```

Comment:
I think this should be `constexpr` or at least `static const`.

Reply:
Will make this static const to include it with internal linkage.  Since this is rarely (if ever) included twice in an executable, it is reasonable to declare it with internal linkage.